src/views/LoginView.vue

```diff
@@ -117,7 +117,7 @@
                 target="_blank"
               >
                 <img class="discord" src="/img/loginView/logo-discord.svg">
-                {{ t('login.joinDiscord') }}
+                <span style="white-space: normal;">{{ t('login.joinDiscord') }}</span>
```

Comment:
For css that doesn't have an existing global Vuetify style, we generally use a class in the html and then select it with css. Can you move this out of sn online style and into the css block? 

Can you also show a screenshot of before and after the change?

Reply:
Okay will be working on it

Reply:
I think it makes more sense to move it outside the media query because we want it to linebreak on any device that needs it based on whatever the button size is. So yes, please move this above

Reply:
@itsalaidbacklife  Made the changes , Please check